addons/project/views/project_views.xml

```diff
@@ -658,7 +658,7 @@
                                                 <div t-if="record.alias_name.value and record.alias_domain.value" class="text-muted">
                                                     <span class="fa fa-envelope-o mr-2" aria-label="Domain Alias" title="Domain Alias"></span><t t-esc="record.alias_id.value"/>
                                                 </div>
-                                                <div t-if="record.rating_active.raw_value" class="text-muted" title="Percentage of happy ratings over the past 30 days." groups="project.group_project_rating">
+                                                <div t-if="record.rating_active.raw_value" class="text-muted" title="Average satisfaction rate for the last 30 days" groups="project.group_project_rating">
```

Comment:
I think we could keep the final point in the tooltip. 🤔

Reply:
Done.